components/automate-ui/src/app/pages/automate-settings/automate-settings.component.ts

```diff
@@ -391,7 +326,10 @@ export class AutomateSettingsComponent implements OnInit, OnDestroy {
 
     switch (job.name) {
       case InfraJobName.MissingNodesForDeletion: {
-        this.handleDisable(this.clientRunsRemoveNodes, job.disabled);
+        if (!this.isDesktopView) {
+          this.handleDisable(this.clientRunsRemoveNodes, job.disabled);
+        }
+
```

Comment:
We only want to modify the disabling the missing node's control if it is not in the desktop view.